### ISSUE TITLE
Serving a file from github pages : javax.net.ssl.SSLException: Received fatal alert: protocol_version

### ISSUE THREAD

**Issue by JirkaDellOro** [plantuml/plantuml-server#93]
Could that be fixed? Test here:

```
@startuml
!includeurl https://jirkadelloro.github.io/FUDGE/Design/Core.txt
@enduml
```

Core.txt files looks like this:
```
@startuml
skinparam monochrome true
skinparam componentStyle uml2
skinparam classAttributeIconSize 0
skinparam shadowing true
skinparam handwritten true

namespace NamespaceTest {
  class A {
      +sayHello(): void
  }
  package "Components" <<Folder>> {
    class B extends NamespaceTest.A {
        -a: A 
        #b:nsdlvkn 
        +sayHello(): void
    }
    class C extends NamespaceTest.A {
        -a: A 
        #b:nsdlvkn 
        +sayHello(): void
    }
  }
}
@enduml
```

Any ideas on how to serve this file from github pages without the exception? The same file runs when served from raw.github, but that's a private repo and the token will keep changing on every  update...

**Comment by arnaudroques** [comment]
Thanks for the feedback.
This is related to https://stackoverflow.com/questions/39157422/how-to-enable-tls-1-2-in-java-7
Should be fixed now. Tell us if it's not working for you!

**Comment by JirkaDellOro** [comment]
Great, that you took care of this that quick. It works now... at least sometimes. It feels like it fails more often, when the same url of the document to render is reused, like a cache-problem. I started to add a # or ? with a random number to the url and it seemed to work more often. However, that's just a feeling, I didn't really measure it.

I also noticed, that the canvas size appears to be limited in the server version, even when rendering to svg. I got myself the plantuml-plugins for VSCode now and I think I'm going to work locally and not try the server any longer.
